compiled_starters/typescript/app/main.ts

```diff
@@ -0,0 +1,19 @@
+import * as dgram from "dgram";
+
+// You can use print statements as follows for debugging, they'll be visible when running tests.
+console.log("Logs from your program will appear here!");
+
+// Uncomment this block to pass the first stage
+//
+// const udpSocket: dgram.Socket = dgram.createSocket("udp4");
+// udpSocket.bind(2053, "127.0.0.1");
+//
+// udpSocket.on("message", (data: Buffer, remoteAddr: dgram.RemoteInfo) => {
+//     try {
+//         console.log(`Received data from ${remoteAddr.address}:${remoteAddr.port}`);
```

Comment:
@ryan-gang can we make this even simpler, maybe just do `console.log(Received...` and not send anything back? That'd help avoid all of this try / catch stuff.

As a side note, the catch block should probably say "error sending data", not "error receiving data".

Reply:
In the first stage the task is to respond back with a UDP packet. So, that part has to stay. 
I have fixed the catch block log.